tests/base/system_info001/NumProcessors.idr

```diff
@@ -1,6 +1,7 @@
 import System.Info
 
 main : IO ()
-main = do np <- getNProcessors
+main = do Just np <- getNProcessors
+            | Nothing => putStrLn "ERROR: Couldn't get number of processors!"
           putStrLn $ (show np) ++ " processors"
```

Comment:
Argh, sorry about that.

Reply:
No problem at all. Thanks for implementing the `Maybe Nat` wrapper! ^^